alf/environments/alf_wrappers.py

```diff
@@ -1049,3 +1050,89 @@ def _scale_back(a, paras):
                                                  self._affine_paras)
         time_step = self._env.step(scaled_action)
         return time_step._replace(prev_action=action)
+
+
+class BatchEnvironmentWrapper(AlfEnvironment):
+    """Wrapper to make a list of non-batched environment into a batched environment.
```

Comment:
Suggest explaining the "batch" behavior is actually sequentially step the environments here.